### PR TITLE
♻️Refactor BPMN-Studio modules

### PR DESCRIPTION
**Pull request by alexanderkasten**
## What did you change?

This PR refactors the typescript part of the bpmn-studio modules.

## How can others test the changes?

- checkout the branch
- `npm run build`
- `npm run electron-start-dev`
- test the studio with our test form https://goo.gl/forms/83UtLkkIIePkAoWI2
and check whether all works as expected

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

File: src/modules/processdef-detail/processdef-detail.ts [hunk 1]
```diff
@@ -221,7 +221,7 @@ export class ProcessDefDetail {
     return this
       ._processEngineService
       .getProcessDefById(this._processId)
-      .then((result: any) => {
+      .then((result: (IProcessDefEntity | any)) => {
```
Comment (SteffenKn) [inline src/modules/processdef-detail/processdef-detail.ts]: Replace any.

File: src/modules/processdef-detail/processdef-detail.ts [hunk 2]
```diff
@@ -314,7 +314,7 @@ export class ProcessDefDetail {
 
     //  Save the diagram to the ProcessEngine {{{ //
     // TODO: Explain what this is doing -> Refactor.
-    let response: any;
+    let response: Response | any;
```
Comment (SteffenKn) [inline src/modules/processdef-detail/processdef-detail.ts]: Replace any.

File: src/modules/processdef-detail/processdef-detail.ts (hunk 1 above)
Comment (alexanderkasten) [reply src/modules/processdef-detail/processdef-detail.ts]: WIth ?

File: src/modules/processdef-detail/processdef-detail.ts (hunk 2 above)
Comment (alexanderkasten) [reply src/modules/processdef-detail/processdef-detail.ts]: With ?

**Pull request merged**